public/fonts/fonts.css

```diff
@@ -30,7 +30,7 @@
 
 @font-face {
     font-family: 'Fira Mono';
-    font-style: normal;
-    font-weight: 500;
+    font-style: normal, bold;
```

Comment:
`font-style` is for italics vs normal, not for bold. Bold is the _weight_ and the keyword `bold` is an alias for `700`, thus only works in the `font-weight` rule.